components/favoris/offres/OffresList.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from 'react'
+
+import OffreRow from 'components/favoris/offres/OffreRow'
+import { HeaderCell } from 'components/rdv/HeaderCell'
+import { Offre } from 'interfaces/favoris'
+
+interface FavorisListProps {
+  offres: Offre[]
+}
+
+const OffresList = ({ offres }: FavorisListProps) => {
```

Comment:
```suggestion
const TableauOffres = ({ offres }: FavorisListProps) => {
```
comme il s'agit bien d'un tableau. ou OffresTableau, sur le projet on a plutôt tendance à mettre en préfixe